### PR TITLE
Fetching Users Info for Crypto

### PR DESCRIPTION
**Pull request by PremanandChowdhury**
Need Suggestions for improvements 🙏🏼 
---
- Issue No. [Link](https://github.com/Real-Dev-Squad/website-crypto/issues/151   "Crypto")

- API Contract : [Link](https://github.com/Real-Dev-Squad/website-api-contracts/blob/28dc5d141c89a7a91cc37769d935dace7e30456b/crypto-site/README.md#getuserinfouser_id "Crypto UserInfo")

---
- Resolving the PR [link](https://github.com/Real-Dev-Squad/website-backend/pull/200 "userinfo Crypto PR")

### CODE REVIEW

**Comment by ankurnarkhede** [comment]
@Kratika0907 , the API contract https://github.com/Real-Dev-Squad/website-api-contracts/blob/28dc5d141c89a7a91cc37769d935dace7e30456b/crypto-site/README.md needs clarifications around the authorization.
Can you add it there?

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]
I am waiting for @Kratika0907 to modify the contract.
In the meanwhile, @PremanandChowdhury , the PR doesn't have unit and integration tests.

**Comment by PremanandChowdhury** [comment]
I'll Check the docs and write the Unit and Integration tests.

File: models/crypto.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* Get all user Information from crypto site
+* @params {userId}
+* @return {Promise<{cryptoUserModel|Object}>}
+*/
+
+const fetchUser = async (userId) => {
+  try {
+    let userInfo
+    if (userId) {
+      const user = await cryptoUserModel.where('user_id', '==', userId).get()
```
[the remark this answers is not in the document]
Comment (PremanandChowdhury) [reply models/crypto.js:20]: Crypto will have separate ``user`` model 👍🏼

[the remark this answers is not in the document]
Comment (swarajpure) [reply models/crypto.js:20]: We're still accessing `users` collection 🤔

[the remark this answers is not in the document]
Comment (PremanandChowdhury) [reply models/crypto.js:20]: For crypto IDK 😇

**Pull request closed without merging**